composer.json

```diff
@@ -23,25 +23,18 @@
   },
   "require": {
     "php": "^7.3",
-    "phpstan/phpstan": "^1.2.0",
+    "phpstan/phpstan": ">=0.12.7 <=1.2.0",
```

Comment:
I'm no composer expert but wont this prevent installing version 1.2.1 and higher?
You can specify it as "^0.12.17|^1.2.0". But maybe it's functionally same as what you wrote, not sure.

Reply:
possible, i don't know how the standard in such cases is, should it be compatible until it breaks?
so we could do >= 0.12.*